controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -828,31 +828,32 @@ def test_get_volume_mappings_success(self):
         mappings = self.svc.get_volume_mappings("volume")
         self.assertEqual(mappings, {'Test_P': '0', 'Test_W': '1'})
 
-    def test_get_first_free_lun_raises_host_not_found_error(self):
+    def test_get_free_lun_raises_host_not_found_error(self):
         self.svc.client.svcinfo.lshostvdiskmap.side_effect = [
             svc_errors.CommandExecutionError('Failed')]
         with self.assertRaises(array_errors.HostNotFoundError):
-            self.svc.get_first_free_lun('host')
+            self.svc.get_free_lun('host')
 
-    def test_get_first_free_lun_with_no_host_mappings(self):
+    def test_get_free_lun_with_no_host_mappings(self):
         self.svc.client.svcinfo.lshostvdiskmap.return_value = []
-        lun = self.svc.get_first_free_lun('host')
-        self.assertEqual(lun, '0')
+        lun = self.svc.get_free_lun('host')
+        self.assertTrue(self.svc.MIN_LUN_NUMBER <= int(lun) <= self.svc.MAX_LUN_NUMBER)
```

Comment:
do we really want to have randomness in our unit tests?
could we mock `choice` instead?

Reply:
done